articles/quickstart/backend/rails/01-authentication.md

```diff
@@ -117,4 +117,45 @@ class SecuredPingController < ActionController::API
     render json: "All good. You only get this message if you're authenticated."
   end
 end
-```
\ No newline at end of file
+```
+
+## Configuring Scopes
+
+The `JsonWebToken.verify` method above verifies that the `access_token` included in the request is valid; however, it doesn't yet include any mechanism for checking that the token has the sufficient **scope** to access the requested resources.
+
+Scopes provide a way for you to define which resources should be accessible by the user holding a given `access_token`. For example, you might choose to permit `read` access to a `messages` resource if a user has a **manager** access level, or a `write` access to that resource if they are an **administrator**.
+
+To configure scopes in your Auth0 dashboard, navigate to [your API](${manage_url}/#/apis) and choose the **Scopes** tab. In this area you can apply any scopes you wish, including one called `read:messages`, which will be used in this example.
+
+## Protect Individual Endpoints
+
+To look for a particular `scope` provide an array of required scopes and check if the required ones are present in the given `access_token` by being part of the `payload`.
```

Comment:
Let's change this paragraph to:


To look for a particular `scope` in an `access_token`, provide an array of required scopes and check if they are present in the payload of the token.